docs/dev-guide/contributing/branching.rst

```diff
@@ -43,7 +43,7 @@ on GitHub. The basic protection that disallows force-push and deletion is the
 only option that should be enabled. There should be no restrictions on required
 status checks or who can push. There is a script at
 `devel/scripts/protected-branches.py
-<https://github.com/pulp/devel/tree/master/scripts/protected-branches.py>`_
+<https://github.com/pulp/devel/blob/master/scripts/protect-branches.py>`_
```

Comment:
the tree--blob change might not be necessary, though https://github.com/tree/devel/blob/master/scripts/protect-branches.py auto-redirects to https://github.com/pulp/devel/blob/master/scripts/protect-branches.py